homeassistant/components/withings/__init__.py

```diff
@@ -0,0 +1,136 @@
+"""
+Support for the Withings API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/withings/
+"""
+import voluptuous as vol
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import HomeAssistantType, ConfigType
+from homeassistant.helpers import config_validation as cv
+from homeassistant.components.withings import (
```

Comment:
All internal integration package imports should be relative to allow custom component override.
```py
from . import ...
```